### DISCUSSION TITLE
feat: event manager move up/down

### DISCUSSION THREAD

**Discussion by brunodavi** [Sketchware-Pro/Sketchware-Pro#474] [Ideas]
Good morning, I had a little problem when organizing the order of the events that to change the order you need to recreate the event

<details>
<summary>Screenshot</summary>

![event_manager](https://user-images.githubusercontent.com/48166556/151366395-3f7afee6-5fc9-4a97-9228-be471604230a.jpg)

> Previously I had created these events in reverse order, to organize this way I had to recreate the onInput.

> So I would like to suggest that you add a way to move the events as well, since this is already done in the blocks

</details>

**Comment by j-px** [comment]
That would definitely be a great feature to add to Sketchware Pro.

**Comment by hassanfayad197123-svg** [comment]
Hi
